scripts/lint-unicode.py

```
@@ -0,0 +1,114 @@
+
+ALLOWED = {0x09, 0x0A, 0x0D}  # TAB, LF, CR
+ALLOW_RE = re.compile(r'bidi-allow:\s*([U+0-9A-Fa-f,]+)')
+
+
+def parse_allow(text):
+    m = ALLOW_RE.search(text)
+    if not m:
+        return frozenset()
+    cps = set()
+    for token in m.group(1).split(','):
+        token = token.strip()
```

Ruff (0.15.20)

[warning] `for` loop variable `token` overwritten by assignment target

(PLW2901)